packages/thirdweb/src/react/web/ui/Bridge/BridgeOrchestrator.tsx

```diff
@@ -113,6 +113,11 @@ export interface BridgeOrchestratorProps {
    * Quick buy amounts
    */
   presetOptions: [number, number, number] | undefined;
+
+  /**
+   * Whether to enable the "Buy with card" (fiat payment) option
+   */
+  enableCard?: boolean;
```

Comment:
What if we do an array instead of litterals

Crypt, card (later we'll have more like ACH, exchange)